src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Listing/Facets.html

```diff
@@ -1,18 +1,15 @@
 <div class="facets">
-    <ul>
-        <li data-ng-repeat="facet in facets" class="facet facet-{{facet.key}}">
-            <h3 class="facet-header">{{ facet.name | translate }}</h3>
-            <ul>
-                <li data-ng-repeat="item in facet.items" class="facet-item-{{item.key}}">
-                    <a
-                        href=""
-                        class="facet-item"
-                        data-ng-class="{'is-active': item.enabled}"
-                        data-ng-click="toggleItem(facet, item, $event)">
-                        {{ item.name | translate }}
-                    </a>
-                </li>
-            </ul>
-        </li>
-    </ul>
-</div><!-- /.facets -->
+    <div data-ng-repeat="facet in facets" class="facet facet-{{facet.key}}">
+        <h3 class="facet-header">{{ facet.name | translate }}</h3>
+        <div class="facet-buttons">
+            <a
+                href=""
+                data-ng-repeat="item in facet.items"
+                class="facet-item-{{item.key}} facet-item button-cta-secondary"
+                data-ng-class="{'is-active': item.enabled}"
+                data-ng-click="toggleItem(facet, item, $event)">
+                {{ item.name | translate }}
+            </a>
+        </div>
+    </div>
+</div>
```

Comment:
Why did you change this? IMHO it has less symantics and no real advantages. See also  5fc7910.